buildSrc/build.gradle.kts

```diff
@@ -57,7 +57,7 @@ dependencies {
   implementation(libs.pluginPackages.taskTree)
   implementation(libs.pluginPackages.versionCatalogUpdate)
   implementation(libs.pluginPackages.versionsFilter)
-  testImplementation(libs.junit)
+  testImplementation(libs.junit.api)
```

Comment:
Huh how come these tests are passing with no (?) junit-engine on the classpath 🤔